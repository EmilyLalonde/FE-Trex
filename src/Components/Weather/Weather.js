import React, { Component } from 'react';
import {
  StyleSheet,
  Text,
  View,
  ScrollView,
  TouchableOpacity,
  TextInput,
  Image
} from "react-native";

export default class Weather extends Component {
  constructor() {
    super() 
    this.state = {
      weather: []
    }
  }

  render() {
    
    return (
      <View style={styles.container}>
        <View style={styles.text}>
        <Text style={styles.weatherText}>Paris</Text>
        <Image  style={{width: 120, height: 110, marginLeft: 30}} source={{uri: 'https://cdn3.iconfinder.com/data/icons/picons-weather/57/15_heavy_rain-512.png'}}/>
        <Text style={styles.weatherText}>54Â°</Text>
        <Text style={styles.weatherText}>Rainy</Text>
        </View>
      </View>
    )
  }
}

const styles = StyleSheet.create({
  text: {
    borderColor: 'white',
    borderWidth: 1,
    borderRadius: 8,
    borderStyle: 'solid',
    color: 'white',
    height: 200,
    marginVertical: 19,
    width: 180,
    marginRight: 10
  },
  weatherText: {
    color: 'white',
    textAlign: 'center',
    fontSize: 18,
    fontWeight: '600',
    marginVertical: 4
  }
});