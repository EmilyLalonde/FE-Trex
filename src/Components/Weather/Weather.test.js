import React from "react";
import "react-native";
import Weather from "./Weather";
import { shallow } from "enzyme";

describe("Weather", () => {
  it("should return true", () => {
    expect(true).toEqual(true);
  });
  it("should match the snapshot when loading", () => {
    let wrapper = shallow(<Weather navigation={{ navigate: jest.fn() }} />);

    expect(wrapper).toMatchSnapshot();
  });
});
