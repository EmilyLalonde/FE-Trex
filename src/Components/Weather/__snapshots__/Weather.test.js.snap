// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weather should match the snapshot when loading 1`] = `
<Text
  style={
    Object {
      "borderColor": "white",
      "borderRadius": 8,
      "borderStyle": "solid",
      "borderWidth": 1,
      "color": "white",
      "fontSize": 20,
      "height": 200,
      "marginVertical": 15,
      "textAlign": "center",
      "width": 200,
    }
  }
>
  Insert Weather Here
</Text>
`;
